WebContent/resources/admin.js

```diff
@@ -427,15 +428,34 @@ $(document).ready( function() {
 	  }
 	});
 	
-	$( "#confirm-update-tabs" ).tabs();
 	
 	
+	$( "#confirm-update-tabs" ).tabs();
 	
-	$("#btUpdateSurvey").button().button().click(function(event) {
-	      event.preventDefault();
-	      updateSurvey();
+	$( "#btUpdateSurvey" ).click(function() {
+			event.preventDefault();
+			var gitversion = $('select.git-select').find(":selected").val();
+			
+			if ( gitversion == "None" ) {
+				$('#UpdateSurveyModal').modal('hide');
```

Comment:
If a git version is not selected, it will default to the latest from the repository.  We don't want to display a warning.